.gitignore

```diff
@@ -5,6 +5,15 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+firebase-debug.log*
+firebase-debug.*.log*
+
+# Firebase config
+
+# Uncomment this if you'd like others to create their own Firebase project.
+# For a team working on the same Firebase project(s), it is recommended to leave
+# it commented so all members can deploy to the same project(s) in .firebaserc.
+# .firebaserc
```

Comment:
Can you put the 2 log lines inside the `# Firebase config` section ?

Reply:
I prefer to keep the log files in `Logs` even if they are linked to firebase and the config files in `Firebase config`